jscomp/super_errors/super_typetexp.ml

```diff
@@ -121,8 +121,8 @@ let report_error env ppf = function
       fprintf ppf "@[<v>\
       @{<info>The record field %a can't be found.@}@,@,\
       If it's defined in another module or file, bring it into scope by:@,\
-      @[- Annotating it with said module name:@ @{<info>let baby = {MyModule.age: 3}@}@]@,\
-      @[- Or specifying its type:@ @{<info>let baby: MyModule.person = {age: 3}@}@]\
+      @[- Prefixing it with said module name:@ @{<info>TheModule.%a@}@]@,\
```

Comment:
same here, you're adding 2 more `%a` pragmas but no arguments on what to print?!